libvips/foreign/cgifsave.c

```diff
@@ -572,10 +665,6 @@ vips_foreign_save_cgif_build( VipsObject *object )
 	frame_rect.top = 0;
 	frame_rect.width = cgif->in->Xsize;
 	frame_rect.height = page_height;
-	if( (guint64) frame_rect.width * frame_rect.height > 5000 * 5000 ) {
```

Comment:
Perhaps we should still check the theoretical dimension limit?:
```c
	/* GIF has a limit of 64k per axis -- double-check this.
	 */
	if( frame_rect.width > 65536 || frame_rect.height > 65536 ) {
```

Reply:
Good point! Added the check